CHANGELOG.rst

```diff
@@ -3,7 +3,8 @@ Changelog
 
 in development
 --------------
-
+* Fixed issue where passing a single integer member for an array parameter for an action would
```

Comment:
Minor thing - we tend to follow chronological order and put latest changes at the end :)